tests/.eslintrc.yml

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+env:
+  jasmine: true
```

Comment:
see https://github.com/apache/cordova-plugin-statusbar/pull/143#pullrequestreview-250956599

Reply:
I was doing for the consistency across other repositories. For example, iOS, Android, Electron has a separate file. As well, it's a good idea to keep configurations global in a separate file, rather than inside a specific test file. Because otherwise, that would mean you would have to go to each test file and add it manually.